encord/group.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Dict, Optional
+from uuid import UUID
+
+import encord.orm.group as orm_group
+from encord.http.v2.api_client import ApiClient
+
+
+class Group:
+    def __init__(self, api_client: ApiClient, orm_group: orm_group.Group):
+        self._api_client = api_client
+        self._orm_group = orm_group
+        self._parsed_metadata: Optional[Dict[str, Any]] = None
+
+    @property
+    def uuid(self) -> str:
```

Comment:
Don't rename the field here; it's called `group_hash` everywhere else.

It's a silly name but it's *the* name